.github/ISSUE_TEMPLATE.md

```diff
@@ -1,17 +1,22 @@
+**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you do not read fill out the issue template repeatedly you will be blocked from every submitting issues to Homebrew again.**
```

Comment:
"If you do not read fill out the issue template repeatedly"

the "do not read" seems out of place

Should it be "If you fill out the issue template repeatedly"?